SKILL.tsv

```diff
@@ -5201,9 +5201,9 @@ SKILL_20150918_005200	$Increases the damage dealt on an enemy with [High Kick] b
 SKILL_20150918_005201	$Earth Wave: Remove Knockdown
 SKILL_20150918_005202	$[Earth Wave] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
 SKILL_20150918_005203	$Steed Charge: Remove Knockdown
-SKILL_20150918_005204	$[Steed Charge] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
+SKILL_20150918_005204	$[Steed Charge] no longer knocks enemies down. (Only applies if the attribute is set to ON)
 SKILL_20150918_005205	$Doom Spike
-SKILL_20150918_005206	$[Doom Spike] no longer knockdowns enemies. (Only applies if the attribute is set to ON)
+SKILL_20150918_005206	$[Doom Spike] no longer knocks enemies down. (Only applies if the attribute is set to ON)
```

Comment:
I have changed almost everything besides Perspective Distortion because it is a direct translation and this one:

https://github.com/Treeofsavior/EnglishTranslation/pull/1302#discussion-diff-43199318R794

I still think it should just be 'Knock the enemy back [...]' because the "Attack" before it isn't "Attacks". If it was "Knocks back" i think it would imply the skill is the one doing both the attacking and the knocking back - not the character.  @ttgmichael thoughts?